src/openrct2/CmdlineSprite.cpp

```diff
@@ -105,7 +106,7 @@ static bool sprite_file_open(const utf8* path)
             return false;
         }
 
-        spriteFileData = static_cast<uint8_t*>(malloc(spriteFileHeader.total_size));
+        spriteFileData = std::make_unique<uint8_t>(spriteFileHeader.total_size).get();
```

Comment:
You allocate and immediately free your memory on this line. You need to save the result of this not discard it